### DISCUSSION TITLE
MacOS Close Windows NOT Close Database

### DISCUSSION THREAD

**Discussion by nmajin** [keepassxreboot/keepassxc#6059] [Ideas]
Almost every MacOS application has the keyboard shortcut of "⌘W" to close the Window, so could this be changed instead of closing the database (current configuration)?

Personally, I like closing the window so the app stays in my status bar and not in my dock, which Hide unfortunately keeps it there.

**Comment by phoerious** [comment]
Can you please repost this is a feature request on the issue tracker? Thanks.

**Reply by nmajin** [reply]
Sure, sorry about that.

**Reply by phoerious** [reply]
Just realised this is already tracked in #2672.
